itc/service/src/main/resources/graphql/ObservationDB.graphql

```diff
@@ -7584,8 +7584,8 @@ type GuideEnvironment {
 
 """Enumeration for Guide Probes"""
 enum GuideProbe {
-  PWFS_1
-  PWFS_2
+  PWFS1
```

Comment:
How is this not flagged

Reply:
I think I edited the wrong file. This one is in ITC, not `modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql`. But I'm not sure if it is still used for anything?